front/app/components/Form/Components/Layouts/CLSurveyPageLayout.tsx

```diff
@@ -242,57 +215,43 @@ const CLSurveyPageLayout = memo(
       scrollToTop();
     };
 
-    if (!project) {
-      return null;
-    }
-
     return (
       <>
         <Box
-          width="100%"
-          height="100%"
-          pt="82px"
-          pb="72px"
-          maxWidth="700px"
           display="flex"
           flexDirection="column"
-          justifyContent="center"
-          alignItems="center"
+          maxWidth="700px"
+          height="100%"
         >
-          <SurveyHeading
-            project={project.data}
-            titleText={localize(
-              phase?.data.attributes.native_survey_title_multiloc
-            )}
-            canUserEditProject={userIsModerator}
-            loggedIn={!isNilOrError(authUser)}
-            percentageAnswered={percentageAnswered}
-          />
```

Comment:
Took out SurveyHeading. It consisted of the buttons and the progress indicator. 
- The progress bar was taken out of SurveyHeading, partly because it's not really related to the rest of the component but also because I got rid of a layout issue after moving it to the bottom (where it may be even better located next to the control buttons) 
- The survey heading was moved to IdeasNewSurveyForm because, especially without the progress bar, it's not related to the core form (it closes the portal/modal and has the Edit survey button) Therefore, it shouldn't be in CLSurveyPageLayout.